tests/parser/functions/test_convert_to_int128.py

```diff
@@ -257,108 +254,13 @@ def foo() -> int128:
     )
 
 
-def test_convert_from_address(w3, get_contract):
+def test_convert_from_address(get_contract, assert_compile_failed):
     code = """
-stor: address
-
 @external
 def testCompiles():
     x: int128 = convert(msg.sender, int128)
-
-@external
```

Comment:
This seems like a useful test, was it moved?

Reply:
since address -> int128 is now banned, we no longer need to check multiple cases of it working, we can just check one case that it is disallowed.